tests/Controller/EnMarche/CitizenProjectControllerTest.php

```diff
@@ -496,9 +496,8 @@ public function testCitizenProjectLandingPageResultAuthenticate()
 
         $thumb3 = $crawler->filter('.citizen-projects__landing__card')->eq(2);
 
-        // This test fails only in CircleCI, need investigation
-//        $this->assertSame('Le projet citoyen à Marseille', trim($thumb3->filter('h3')->text()));
```

Comment:
Is the comment still needed?

Reply:
No, the data was changed, there is no more this CP in test result.

Reply:
So please remove it.